### ISSUE TITLE
Suggest to add a WebViewJavascriptBridgeProtocol like this 

### ISSUE THREAD

**Issue by aelam** [marcuswestin/WebViewJavascriptBridge#203]
People can use the the same bridge with `id<WebViewJavascriptBridgeProtocol> bridge` while they have an UIWebView and a WKWebView 

```
#import <Foundation/Foundation.h>

@protocol WebViewJavascriptBridgeProtocol <NSObject>

+ (instancetype)bridgeForWebView:(id)webView;
+ (void)enableLogging;
+ (void)setLogMaxLength:(int)length;

- (void)registerHandler:(NSString*)handlerName handler:(WVJBHandler)handler;
- (void)callHandler:(NSString*)handlerName;
- (void)callHandler:(NSString*)handlerName data:(id)data;
- (void)callHandler:(NSString*)handlerName data:(id)data responseCallback:(WVJBResponseCallback)responseCallback;
- (void)setWebViewDelegate:(id)webViewDelegate;

@end
```

Then it's possible to invoke like 

```
id < WKWebViewJavascriptBridgeProtocol> bridge = [FixedNameBridge bridgeForWebView:xWebView];
```

`FixedNameBridge` could be an UtilClass 
`xWebView` can be an `UIWebView` or a `WKWebView`

rather than 

```
        if (_wkWebView) {
            self.bridge = [WKWebViewJavascriptBridge bridgeForWebView:_wkWebView];
            [self.bridge setWebViewDelegate:self];
        } else {
            self.bridge = [WebViewJavascriptBridge bridgeForWebView:_webView];
            [self.bridge setWebViewDelegate:self];
        }
```

**Comment by marcuswestin** [comment]
Nice! I'd definitely accept a PR with this

**Comment by marcuswestin** [comment]
This is fixed as of f3d2aceb5bda
